processing/src/main/java/org/apache/carbondata/processing/util/CarbonDataProcessorUtil.java

```diff
@@ -145,6 +147,15 @@ public static void renameBadRecordsFromInProgressToNormal(
     }
   }
 
+  public static void renameBadRecord(CarbonDataLoadConfiguration configuration) {
+    // rename the bad record in progress to normal
+    CarbonTableIdentifier identifier =
+        configuration.getTableIdentifier().getCarbonTableIdentifier();
+    CarbonDataProcessorUtil.renameBadRecordsFromInProgressToNormal(configuration,
```

Comment:
1. Avoid calling the method using classname. Method is present in the same class.
2. I think we can introduce a new class called BadRecordsUtil which will have all the functions related to bad records like 
renameBadRecord, hasBadRecord, renameBadRecordsFromInProgressToNormal (make it private in the new util class)

Reply:
Refactored